src/parse/GrammarAncestry.sml

```diff
@@ -26,7 +26,7 @@ fun ancestry {thy} =
                     NONE => raise ERR "ancestry" "Badly encoded data"
                   | SOME sl => sl
 
-fun set_ancestry sl =
+fun from_ancestry sl =
```

Comment:
This one probably shouldn't be changed? Because it actually does a stateful update, I think.

Reply:
Indeed,thank you